### ISSUE TITLE
[Bug]: occ outputs errors on 26beta5

### ISSUE THREAD

**Issue by jospoortvliet** [nextcloud/maps#976]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Maps app **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

testcloud:/var/www/tech-preview.nextcloud.com# sudo -u www-data php -d memory_limit=4G ./occ status
An unhandled exception has been thrown:
Error: Class "OCP\AppFramework\Db\Mapper" not found in /var/www/tech-preview.nextcloud.com/apps/maps/lib/DB/GeophotoMapper.php:18
Stack trace:
#0 /var/www/tech-preview.nextcloud.com/lib/composer/composer/ClassLoader.php(578): include()
#1 /var/www/tech-preview.nextcloud.com/lib/composer/composer/ClassLoader.php(432): Composer\Autoload\ClassLoader::Composer\Autoload\{closure}()
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass()
#3 /var/www/tech-preview.nextcloud.com/lib/private/AppFramework/Utility/SimpleContainer.php(122): ReflectionClass->__construct()
#4 /var/www/tech-preview.nextcloud.com/lib/private/AppFramework/Utility/SimpleContainer.php(142): OC\AppFramework\Utility\SimpleContainer->resolve()
#5 /var/www/tech-preview.nextcloud.com/lib/private/AppFramework/DependencyInjection/DIContainer.php(493): OC\AppFramework\Utility\SimpleContainer->query()
#6 /var/www/tech-preview.nextcloud.com/lib/private/AppFramework/DependencyInjection/DIContainer.php(465): OC\AppFramework\DependencyInjection\DIContainer->queryNoFallback()
#7 /var/www/tech-preview.nextcloud.com/lib/private/AppFramework/Utility/SimpleContainer.php(97): OC\AppFramework\DependencyInjection\DIContainer->query()
#8 [internal function]: OC\AppFramework\Utility\SimpleContainer->OC\AppFramework\Utility\{closure}()
#9 /var/www/tech-preview.nextcloud.com/lib/private/AppFramework/Utility/SimpleContainer.php(116): array_map()
#10 /var/www/tech-preview.nextcloud.com/lib/private/AppFramework/Utility/SimpleContainer.php(124): OC\AppFramework\Utility\SimpleContainer->buildClass()
#11 /var/www/tech-preview.nextcloud.com/lib/private/AppFramework/Utility/SimpleContainer.php(142): OC\AppFramework\Utility\SimpleContainer->resolve()
#12 /var/www/tech-preview.nextcloud.com/lib/private/AppFramework/DependencyInjection/DIContainer.php(493): OC\AppFramework\Utility\SimpleContainer->query()
#13 /var/www/tech-preview.nextcloud.com/lib/private/AppFramework/DependencyInjection/DIContainer.php(465): OC\AppFramework\DependencyInjection\DIContainer->queryNoFallback()
#14 /var/www/tech-preview.nextcloud.com/lib/private/AppFramework/Utility/SimpleContainer.php(97): OC\AppFramework\DependencyInjection\DIContainer->query()
#15 [internal function]: OC\AppFramework\Utility\SimpleContainer->OC\AppFramework\Utility\{closure}()
#16 /var/www/tech-preview.nextcloud.com/lib/private/AppFramework/Utility/SimpleContainer.php(116): array_map()
#17 /var/www/tech-preview.nextcloud.com/lib/private/AppFramework/Utility/SimpleContainer.php(124): OC\AppFramework\Utility\SimpleContainer->buildClass()
#18 /var/www/tech-preview.nextcloud.com/lib/private/AppFramework/Utility/SimpleContainer.php(142): OC\AppFramework\Utility\SimpleContainer->resolve()
#19 /var/www/tech-preview.nextcloud.com/lib/private/AppFramework/DependencyInjection/DIContainer.php(493): OC\AppFramework\Utility\SimpleContainer->query()
#20 /var/www/tech-preview.nextcloud.com/lib/private/ServerContainer.php(155): OC\AppFramework\DependencyInjection\DIContainer->queryNoFallback()
#21 /var/www/tech-preview.nextcloud.com/lib/private/Console/Application.php(221): OC\ServerContainer->query()
#22 /var/www/tech-preview.nextcloud.com/lib/private/Console/Application.php(131): OC\Console\Application->loadCommandsFromInfoXml()
#23 /var/www/tech-preview.nextcloud.com/console.php(99): OC\Console\Application->loadCommands()
#24 /var/www/tech-preview.nextcloud.com/occ(11): require_once('...')

-> after installing the Maps app on Nextcloud Hub 4 beta 5 (26.0.0.7). Disabling the Maps apps makes oCC work again.

The good news: the UI is not totally broken or something, the app works. Found some minor issues (dragging map often breaks, can't scroll in route information as it instead zooms the map, clicking on route information items moves the route image around in weird ways etc) but the basics work.

### Steps to reproduce

1. Install Maps
2. Run occ command
3. Enjoy output

### Expected behavior

Less exciting output

### Installation method

None

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install of the server?

None

### Is this bug present after an update or on a fresh install of the app?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### List of activated Apps

```shell
Enabled:
  - activity: 2.18.0
  - admin_audit: 1.16.0
  - analytics: 4.7.2
  - approval: 1.0.12
  - bruteforcesettings: 2.6.0
  - calendar: 4.3.0-alpha.1
  - calendar_resource_management: 0.4.0
  - circles: 26.0.0-beta2
  - cloud_federation_api: 1.9.0
  - comments: 1.16.0
  - contacts: 5.1.0
  - contactsinteraction: 1.7.0
  - dashboard: 7.6.0
  - dav: 1.25.0
  - documentserver_community: 0.1.13
  - event_update_notification: 2.0.0
  - external: 5.1.0
  - federatedfilesharing: 1.16.0
  - federation: 1.16.0
  - files: 1.21.1
  - files_accesscontrol: 1.16.0
  - files_automatedtagging: 1.16.0
  - files_downloadactivity: 1.15.0
  - files_external: 1.18.0
  - files_lock: 26.0.0
  - files_pdfviewer: 2.7.0
  - files_retention: 1.15.0
  - files_rightclick: 1.5.0
  - files_sharing: 1.18.0
  - files_trashbin: 1.16.0
  - files_versions: 1.19.1
  - firstrunwizard: 2.15.0
  - flow_notifications: 1.6.0
  - forms: 3.1.0
  - guests: 2.3.0
  - imageconverter: 1.3.4
  - impersonate: 1.11.0
  - integration_github: 2.0.0
  - integration_gitlab: 1.0.12
  - integration_mastodon: 2.0.0
  - integration_mattermost: 1.0.4
  - integration_openproject: 2.2.1
  - integration_zammad: 2.0.3
  - logreader: 2.11.0
  - lookup_server_connector: 1.14.0
  - mail: 2.3.0-alpha.4
  - nextcloud_announcements: 1.15.0
  - notes: 4.6.0
  - notifications: 2.14.0
  - oauth2: 1.14.0
  - password_policy: 1.16.0
  - photos: 2.2.0
  - picker: 1.0.4
  - privacy: 1.10.0
  - provisioning_api: 1.16.0
  - recognize: 3.5.0
  - recommendations: 1.5.0
  - related_resources: 1.1.0-alpha1
  - richdocuments: 8.0.0-beta.1
  - serverinfo: 1.16.0
  - settings: 1.8.0
  - sharebymail: 1.16.0
  - sharepoint: 1.12.0
  - spreed: 16.0.0-beta.1
  - support: 1.9.0
  - survey_client: 1.14.0
  - systemtags: 1.16.0
  - tables: 0.2.2
  - talk_matterbridge: 1.25.2-1
  - tasks: 0.14.5
  - text: 3.7.1
  - theming: 2.1.1
  - twofactor_backupcodes: 1.15.0
  - twofactor_totp: 8.0.0-alpha.0
  - twofactor_webauthn: 1.1.1
  - updatenotification: 1.16.0
  - user_status: 1.6.0
  - viewer: 1.10.0
  - weather_status: 1.6.0
  - welcome: 1.0.8
  - workflow_pdf_converter: 1.10.0
  - workflow_script: 1.10.0
  - workflowengine: 2.8.0
```

**Comment by tacruc** [comment]
@jospoortvliet Thanks for reporting the Issue. Did you install the last stable or the nightly builds?

Looks to me related to https://github.com/nextcloud/maps/issues/948 fixed in the nightly builds.

**Comment by tacruc** [comment]
The fix should be included in [v0.2.5-5-nightly](https://github.com/nextcloud/maps/releases/tag/v0.2.5-5-nightly) and later

**Comment by tacruc** [comment]
I will close this, as v0.2.4 also officially does not support NC26.

Or is this reproducible with v0.2.5?
